packages/types/src/Moment.spec.ts

```diff
@@ -14,7 +14,7 @@ describe('Moment', () => {
         expect(new Moment(input)[toJSON ? 'toJSON' : 'toString']()).toBe(expected);
       });
 
-    testDecode('Date', new Date(1537968546280), '2018-09-26T13:29:07.000Z');
+    testDecode('Date', new Date(1537968546280), 'Wed Sep 26 2018 15:29:07 GMT+0200 (Central European Summer Time)');
```

Comment:
So:
- if we keep the Date's default toString, it will show like this (i.e. with timezone, not sure if this test actually passes in other timezones)
- or we can overwrite toString in Moment to return toIsoString, but users might be confused.

I went with the 1st solution

Reply:
Travis answered my question, it doesn't pass in other timezones. I'll put toISOString then

Reply:
Travis answered me: tests don't pass in other timezones. Only checking the day in tests.